<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Searcher</class>
 <widget class="QWidget" name="Searcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="l_levels">
     <property name="text">
      <string>Уровни</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_levels"/>
   </item>
   <item>
    <widget class="QLabel" name="l_search">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_text"/>
   </item>
   <item>
    <widget class="QToolButton" name="b_search">
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="b_update">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
